dev-qt/qtgui/qtgui-5.15.2-r10.ebuild

```diff
@@ -0,0 +1,184 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+KDE_ORG_COMMIT=7e3f3b0c1f7f417e52ca374228f5afb824a913d6
+QT5_MODULE="qtbase"
+inherit qt5-build
+
+DESCRIPTION="The GUI module and platform plugins for the Qt5 framework"
+SLOT=5/$(ver_cut 1-3) # bug 707658
+
+if [[ ${QT5_BUILD_TYPE} == release ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ppc ~ppc64 ~riscv ~sparc ~x86"
+fi
+
+IUSE="accessibility dbus egl eglfs evdev +gif gles2-only ibus jpeg
```

Comment:
gif doesn't cause a dependency? But I don't think this really matters.

Reply:
no, the gif decoder doesn't have any external dep

Reply:
hm we could consider building it unconditionally, since it doesn't save anything (except probably minor amount of kb)